scrapy/http/request/form.py

```diff
@@ -44,7 +45,7 @@ def from_response(cls, response, formname=None, formid=None, formnumber=0, formd
 
 def _get_form_url(form, url):
```

Comment:
we can pretty much remove this function and do `url = urljoin(form.base_url, kwargs.pop('url', form.action))`